packages/daemon/test/controller/job.test.ts

```diff
@@ -360,4 +383,147 @@ describe('test job controller', () => {
       })
       .expect(404);
   });
+  it('should show the param infomation', () => {
+    app.mockClassFunction('pipelineService', 'getJobById', async (id: string) => {
+      assert.equal(id, 'jobId');
+      return mockJob;
+    });
+
+    return app
+            .httpRequest()
```

Comment:
We should keep a consistent style, see https://github.com/alibaba/pipcook/blob/2d63418a167102088e499ee7ba4ee7d4743834ac/packages/daemon/test/controller/job.test.ts#L83-L90